src/scp/SCPDriver.h

```diff
@@ -63,6 +64,11 @@ class SCPEnvelopeWrapper : public NonMovableOrCopyable
 
 typedef std::shared_ptr<SCPEnvelopeWrapper> SCPEnvelopeWrapperPtr;
 
+// values used to switch hash function between priority and neighborhood checks
```

Comment:
I wasn't sure whether to keep these here or to move them to `HerderSCPDriver` (and by extension to `SCPUnitTests.cpp`).

They're trivial enough that they could be considered noise in here. Let me know what you think.